configure.py

```diff
@@ -678,6 +678,7 @@ def find_headers(repodir, excluded_dirs):
                 'gms/application_state.cc',
                 'gms/inet_address.cc',
```

Comment:
Please reserve the word "fix" for bug fixes, and use a different word for the pull request title and the changelogs, to reduce confusion.

Reply:
I will.